ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -98,6 +98,8 @@ class DemoListViewController: DemoTableViewController {
         let demo = DemoControllerSection.allCases[indexPath.section].rows[indexPath.row]
         cell.setup(title: demo.title, accessoryType: .disclosureIndicator)
         cell.titleNumberOfLinesForLargerDynamicType = 2
+        cell.backgroundStyleType = .grouped
```

Comment:
Was your fluent2-branch updated to latest? I think I already merged this in last week: https://github.com/microsoft/fluentui-apple/pull/1416/files#diff-2792d90e0d200bfb9f58fd1846eb8141042fe4926b240a344893ef29038eae48R101

Reply:
Nvm please ignore I misread fluent2-colors, thought it was the tokens branch!